proto/tendermint/types/types.pb.go

```diff
@@ -422,10 +422,10 @@ type Data struct {
 	// This means that block.AppHash does not include these txs.
 	Txs [][]byte `protobuf:"bytes,1,rep,name=txs,proto3" json:"txs,omitempty"`
 	// field number 2 is reserved for intermediate state roots
-	Evidence           EvidenceList `protobuf:"bytes,3,opt,name=evidence,proto3" json:"evidence"`
-	Messages           Messages     `protobuf:"bytes,4,opt,name=messages,proto3" json:"messages"`
-	OriginalSquareSize uint64       `protobuf:"varint,5,opt,name=original_square_size,json=originalSquareSize,proto3" json:"original_square_size,omitempty"`
-	Hash               []byte       `protobuf:"bytes,6,opt,name=hash,proto3" json:"hash,omitempty"`
+	Evidence   EvidenceList `protobuf:"bytes,3,opt,name=evidence,proto3" json:"evidence"`
+	Messages   Messages     `protobuf:"bytes,4,opt,name=messages,proto3" json:"messages"`
+	SquareSize uint64       `protobuf:"varint,5,opt,name=original_square_size,json=squareSize,proto3" json:"square_size,omitempty"`
+	Hash       []byte       `protobuf:"bytes,6,opt,name=hash,proto3" json:"hash,omitempty"`
```

Comment:
we should never be editing these files, instead, we need to change the proto file and then regenerate the bindings using `make proto-gen`